src/tests/multihost/ipa/test_subid_ranges.py

```diff
@@ -0,0 +1,117 @@
+    """
+    def test_subid_feature(self, multihost):
+        """
+        :Title: support subid ranges managed by FreeIPA
+        :id: 50bcdc28-00c8-11ec-bef4-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1803943
+        :steps:
+            1. Generate subid for user admin
+            2. Test newuidmap command
+            3. Test newgidmap command
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+        """
+        ssh1 = SSHClient(multihost.client[0].ip,
+                         username=user, password=test_password)
+        (result, result1, exit_status) = ssh1.exec_command(f"ipa  "
+                                                           f"subid-find"
+                                                           f"  --owner  "
+                                                           f"{user}")
+        user_details = result1.readlines()
+        uid_start = int(user_details[5].split(': ')[1].split('\n')[0])
+        uid_range = int(user_details[6].split(': ')[1].split('\n')[0])
+        gid_start = int(user_details[7].split(': ')[1].split('\n')[0])
+        gid_range = int(user_details[8].split(': ')[1].split('\n')[0])
+        (results1, results2, results3) = ssh1.exec_command("unshare"
+                                                           " -U bash"
+                                                           " -c 'echo"
+                                                           " $$ > "
+                                                           "/tmp/unshare.pid;"
+                                                           " sleep 100'")
+        time.sleep(2)
+        proces_id = [int(i)
+                     for i in
+                     execute_cmd(multihost,
+                                 "cat /tmp/unshare.pid").stdout_text.split()
+                     if i.isdigit()][0]
+        uid = 0
+        count = 1
+        ssh1.exec_command(f"newuidmap {proces_id} {uid} {uid_start} {count}")
+        ssh1.exec_command(f"newgidmap {proces_id} {uid} {gid_start} {count}")
+        result = execute_cmd(multihost, f"cat /proc/{proces_id}/uid_map")
+        assert str(uid) in result.stdout_text
+        assert str(uid_start) in result.stdout_text
+        assert str(count) in result.stdout_text
+        result = execute_cmd(multihost, f"cat /proc/{proces_id}/gid_map")
+        assert str(uid) in result.stdout_text
+        assert str(gid_start) in result.stdout_text
+        assert str(count) in result.stdout_text
+        multihost.client[0].run_command(f'kill -9 {proces_id}')
+        ssh1.close()
+
+    def test_list_subid_ranges(self, multihost):
+        """
+        :Title: support subid ranges managed by FreeIPA
+        :id: 4ab33f84-00c8-11ec-ad91-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1803943
+        :steps:
+            1. Configure subid: sss on /etc/nsswitch.conf
```

Comment:
IIUC, this is done in class fixture, not it this test, so this comment isn't correct anymore.

Reply:
Fixed